ui/partials/main/thGlobalTopNavPanel.html

```diff
@@ -80,17 +80,60 @@
         </span>
       </span>
 
-      <!-- Global Filter Button -->
-      <span class="btn btn-view-nav btn-right-navbar nav-menu-btn"
-            title="Global job filters"
-            ng-class="{'active': (isFilterPanelShowing)}"
-            ng-click="setFilterPanelShowing(!isFilterPanelShowing)"
-            tabindex="0"
-            role="button"><span>Filters</span>
-        <i class="fa fa-angle-down lightgray"
-           ng-hide="isFilterPanelShowing"></i>
-        <i class="fa fa-angle-up lightgray"
-           ng-show="isFilterPanelShowing"></i>
+      <!-- Filters Menu -->
+      <span ng-controller="FilterPanelCtrl" >
+        <span th-checkbox-dropdown-container class="dropdown">
+          <button id="filterLabel" title="Set filters" role="button"
+                  data-toggle="dropdown"
+                  class="btn btn-view-nav btn-right-navbar nav-menu-btn">Filters
+            <span class="fa fa-angle-down lightgray"></span>
+          </button>
+          <ul id="filter-dropdown"
+              class="dropdown-menu checkbox-dropdown-menu"
+              role="menu" aria-labelledby="filterLabel">
+            <span class="filtergroup" ng-repeat="(group_order, group) in filterGroups">
+              <li role="presentation" class="dropdown-header">
+                  {{::group.name}}
+                  <span class="fa fa-info-circle"></span>
+              </li>
+              <span ng-repeat="filterName in group.resultStatuses" >
+                <label class="checkbox-inline {{::checkClass}} dropdown-item">
```

Comment:
The `checkbox-inline` class listed here no longer has any references in CSS, so can be removed if not needed :-)